tests/conftest.py

```diff
@@ -49,6 +49,7 @@ def task_config(request):
 def model_configs(evaluation_config, task_config):
     model_id_mapping = {
         "sentiment-classification": ["pin/senda"],
+        "discourse-coherence-classification": ["ajders/ddisco_classifier"],
```

Comment:
Should be changed when model has been moved.

Also, I've attempted to be systematic in the naming of models. All the names of our other models follow the form "<language>-<task>-<size>", so maybe you could change it to "da-discourse-coherence-base"?

```suggestion
        "discourse-coherence-classification": ["alexandrainst/da-discourse-coherence-base"],
```